src/de/animestream/build.gradle

```diff
@@ -3,17 +3,11 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlinx-serialization'
 
 ext {
-    extName = 'AnimeStream'
+    extName = 'Anime-Stream'
     pkgNameSuffix = 'de.animestream'
     extClass = '.AnimeStream'
     extVersionCode = 1
```

Comment:
bump

Reply:
Is it the same site with the same library, but just with new layout and url? If so, I don't see why you'd need to remove the old and create a new extension